src/OgAccess.php

```diff
@@ -184,16 +206,16 @@ public function userAccess(EntityInterface $group, $operation, AccountInterface
   public function userAccessEntity($operation, EntityInterface $entity, AccountInterface $user = NULL) {
     $result = AccessResult::neutral();
 
-    // Entity isn't saved yet.
-    if ($entity->isNew()) {
-      return $result->addCacheableDependency($entity);
-    }
-
     $entity_type = $entity->getEntityType();
     $entity_type_id = $entity_type->id();
     $bundle = $entity->bundle();
 
-    if (Og::isGroup($entity_type_id, $bundle)) {
+    if ($this->groupManager->isGroup($entity_type_id, $bundle)) {
+      // Entity isn't saved yet.
+      if ($entity->isNew()) {
+        return $result->addCacheableDependency($entity);
+      }
```

Comment:
Here it is, it has no information about why this is important: https://github.com/amitaibu/og/blob/8.x-1.x/tests/src/Unit/OgAccessEntityTest.php#L39